### ISSUE TITLE
Put id into field value

### ISSUE THREAD

**Issue by rjd22** [zombor/Auto-Modeler#13]
I'm trying to get information from sub tables. While it throws no errors it load the ID of the row into the field value:

<?php echo $anime->type->type; ?>

The result object:

```
object Model_Type(10) {
    protected _table_name => string(5) "types"
    protected _data => array(2) (
        "id" => NULL
        "type" => string(1) "1"
    )
    protected _rules => array(1) (
        "type" => array(1) (
            0 => string(8) "required"
        )
    )
    protected _has_many => array(0) 
    protected _belongs_to => array(0) 
    protected _callbacks => array(0) 
    protected _validation => array(0) 
    protected _validated => bool FALSE
    protected _lang => string(11) "form_errors"
    protected _db => object Database_MySQL(6) {
        protected _connection_id => string(40) "******"
        protected _identifier => string(1) "`"
        public last_query => string(51) "SELECT `id` AS `type` FROM `types` WHERE `id` = '1'"
        protected _instance => string(7) "default"
        protected _connection => resource(mysql link)
        protected _config => array(6) (
            "type" => string(5) "mysql"
            "connection" => array(3) (
                "hostname" => string(24) "*******"
                "database" => string(13) "*********"
                "persistent" => bool FALSE
            )
            "table_prefix" => string(0) ""
            "charset" => string(4) "utf8"
            "caching" => bool TRUE
            "profiling" => bool TRUE
        )
    }
}
```

**Comment by zombor** [comment]
try the develop branch. It will be merged as 3.4 soon

**Comment by zombor** [comment]
This is either resolved or invalid :)
